app/src/main/res/drawable/thumb_23andme.xml

```diff
@@ -0,0 +1,50 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="200dp"
+    android:height="136.85dp"
+    android:viewportWidth="200"
+    android:viewportHeight="136.85">
+  <path
+      android:pathData="M10.96,116.7c-1.73,0 -3.39,-0.4 -4.94,-1.18c-2.61,-1.32 -4.56,-3.58 -5.47,-6.37c-0.91,-2.78 -0.69,-5.76 0.64,-8.37l14.69,-29.93l9.36,30.7l-4.49,9.15C18.87,114.39 15.12,116.7 10.96,116.7z"
+      android:fillColor="#7AC043"
+      android:fillType="evenOdd"/>
```

Comment:
all occurrences of `android:fillType="evenOdd"` are not necessary and can be removed